bluebottle/events/models.py

```diff
@@ -0,0 +1,196 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from djchoices.choices import ChoiceItem
+
+from bluebottle.activities.models import Activity, Contribution
+from bluebottle.notifications.decorators import transition
+
+
+class Event(Activity):
+    start = models.DateTimeField(_('start'))
+    end = models.DateTimeField(_('end'))
+    registration_deadline = models.DateTimeField(_('registration deadline'))
+    capacity = models.PositiveIntegerField()
+    automatically_accept = models.BooleanField(default=True)
```

Comment:
We should check with @Rolfertjan which fields we need, I  am pretty sure the `automatically accept` field should be there. And possibly the `registration_deadline` to.